templates/chart/base.twig

```diff
@@ -80,6 +79,43 @@ dw.backend.ready(function() {
         }
     });
 
+    $('a#duplicate-chart').on('click', function(e){
+        e.preventDefault();
+        $('i#copying-chart').addClass('fa-spinner fa-spin');
+        $.ajax({
+            type: "POST",
+            url: '/api/charts/'+chart.get('id')+'/copy',
+            xhrFields: {
+                withCredentials: true
+            },
+            success: moveChart,
```

Comment:
is `move chart` even needed? if admins copy, it's [sent to my-charts anyway](https://github.com/datawrapper/datawrapper/blob/master/lib/core/build/classes/datawrapper/ChartQuery.php#L114-L120)